docs/repositories-configure/integrations/jira-integration.md

```diff
@@ -24,7 +24,7 @@ To enable the Jira integration:
     -   **API token:** [Jira API token](https://support.atlassian.com/atlassian-account/docs/manage-api-tokens-for-your-atlassian-account/#Create-an-API-token) for the user account that Codacy will use to create new issues in Jira. The Jira integration only supports HTTP basic authentication.
 
     !!! important
-        We recommend that you use a dedicated service account for integrating Codacy with Jira. Jira issues created by Codacy will appear as being reported by this user account.
+        Use a dedicated service account to integrate Codacy with Jira. Jira issues created by Codacy will appear as being reported by this account.
```

Comment:
This is the only place where I'd say that avoiding "we recommend" changes the meaning a bit. This is an optional thing that customers **should** do.

This could be improved by changing the admonition type to `tip`, because it would convey that this is a "nice thing to have".

Reply:
Sounds good. Let's try using a tip here